doc/client_lua_api.md

```diff
@@ -314,7 +314,8 @@ Minetest namespace reference
 Call these functions only at load time!
 
 * `minetest.register_globalstep(function(dtime))`
-    * Called every client environment step, usually interval of 0.1s
+    * Called every server step, usually interval of 0.1s.
```

Comment:
"client environment step" not "server step"

Reply:
Oh sorry, I missed that point in my change suggestion.